wcivf/apps/elections/templates/elections/includes/_post_meta_title.html

```diff
@@ -1 +1,5 @@
-{{ object.election.name }}: The {{ person_posts|length }} candidates in {{ object.post.label }}
\ No newline at end of file
+{% if postelection.people|length %}
```

Comment:
I've noticed these same changes should be applied in the `_post_meta_description.html` include also - might be worth adding in to this PR? https://github.com/DemocracyClub/WhoCanIVoteFor/blob/master/wcivf/apps/elections/templates/elections/includes/_post_meta_description.html#L1